cpp/data/eigerfan/src/eigerfan_main.cpp

```diff
@@ -160,8 +160,8 @@ int parse_arguments(int argc, char** argv, EigerFanConfig &cfg)
           "Set the port to accept control messages on")
       ("addr,s", po::value<std::string>()->default_value(EigerFanDefaults::DEFAULT_STREAM_ADDRESS),
           "Set the address of the stream to connect to")
-      ("sockets,z", po::value<unsigned int>()->default_value(EigerFanDefaults::DEFAULT_NUM_SOCKETS),
-          "Set the number of zmq sockets to connect to the Eiger with")
+      ("contextthreads,z", po::value<unsigned int>()->default_value(EigerFanDefaults::DEFAULT_NUM_CONTEXT_THREADS),
```

Comment:
Can we use `--context-threads`? It would be easier to read.

Reply:
Agreed - change made